src/app/admin-page/student-list/student-list.component.html

```diff
@@ -0,0 +1,13 @@
+<h1>Student list</h1>
+<br>
+<div class="list-group" *ngFor="let student of students$ | async">
+  <a href="#" class="list-group-item list-group-item-action" aria-current="true">
```

Comment:
```suggestion
  <a class="list-group-item list-group-item-action" aria-current="true">
```
href should be not needed :)